### DISCUSSION TITLE
Minimal / IoT image

### DISCUSSION THREAD

**Discussion by LukeMech** [Joshua-Riek/ubuntu-rockchip#983] [Ideas]
Hi, how about building armbian-like minimal ISOs? Or at least providing a build script / instructions, to convert server edition into the minimal one?

I think it will be useful, as the server edition is ~0,8GB, when the minimal one could possibly be <0.3GB and contain only required packages

**Comment by Joshua-Riek** [comment]
About 400MB of the compressed server image is actually firmware required by the kernel. To create a true minimal image I would need to create a new kernel with only the necessary drivers enabled. It would likely take a lot of time to figure out the exact drivers required for specific boards. 

There is a build configuration Ubuntu provides for this, but again this would require a lot of kernel development. If anyone would like to assist with tweaking the kernel maybe this can be done.

**Reply by LukeMech** [reply]
Ouch, didn't know its that much of work to be done :I sadly I'm not any kernel dev, so can't help you right now, but thanks for reply!
